Makefile

```diff
@@ -8,6 +8,15 @@ setup:
 	bundle check || bundle install
 	npm install
 
+lint-js:
+	npm run lint
+
+lint-assets:
+	@npm run optimize-assets > /dev/null
+	@[[ -z `git status assets/img -s` ]] || (echo "Error: Optimize SVG images using 'npm run optimize-assets'"; exit 1)
```

Comment:
```
/bin/sh: 1: [[: not found
```

CircleCI got mad at this

Reply:
> ```
> /bin/sh: 1: [[: not found
> ```
> 
> CircleCI got mad at this

Yea, guess it's not available in Circle 🤷  Eventually to a simpler option using `git diff` in be7c54f6255212141e4952ef5bcb7ee37a757557.